akka-stream/src/main/scala/akka/stream/scaladsl/Sink.scala

```diff
@@ -252,4 +253,22 @@ object Sink extends SinkApply {
   def actorSubscriber[T](props: Props): Sink[T, ActorRef] =
     new Sink(new ActorSubscriberSink(props, DefaultAttributes.actorSubscriberSink, shape("ActorSubscriberSink")))
 
+  /**
+   * Combine several sinks with fun-out strategy like `Broadcast` or `Balance` and returns `Sink`.
+   */
+  def combine[T, U](sinks: Sink[U, _]*)(strategy: Int ⇒ Graph[UniformFanOutShape[T, U], Unit]): Sink[T, Unit] =
```

Comment:
To enforce that it is not used with less than 2, this should (IMO) be:

~~~scala
def combine[T, U](sink1: Sink[U, _], sink2:Sink[U, _], sinkN: Sink[U, _]*)(strategy: Int ⇒ Graph[UniformFanOutShape[T, U], Unit]): Sink[T, Unit] = …
~~~